gitlab/__init__.py

```diff
@@ -30,7 +30,7 @@
 from gitlab import utils  # noqa
 
 __title__ = "python-gitlab"
-__version__ = "2.2.0"
+__version__ = "2.2.1"
```

Comment:
Please leave the version number as it is for now.

```suggestion
__version__ = "2.2.0"
```